packages/website/docs/components/patterns/AI Writing Assistant/main.css

```diff
@@ -0,0 +1,30 @@
+body {
+	background-color: var(--sapBackgroundColor);
+	height: 300px;
+}
+
+.hide {
+	display: none;
+}
+
+.show {
+	display: block;
+}
+
+.icon {
+	color: #1969DA;
+}
+
+.icon-clicked {
+	border-left: 1px solid #1969DA;
+	background-color: white;
+}
+
+.loading-input {
+	background-color: #e1f4ff;
```

Comment:
Make sure the sample will look fine in all themes, usually with hardcoded colors the sample may look broken.